package.json

```diff
@@ -19,8 +19,7 @@
 		"npm": ">=6.9.0 <7"
 	},
 	"config": {
-		"GUTENBERG_PHASE": 2,
-		"COMPONENT_SYSTEM_PHASE": 1
```

Comment:
The only instance of `COMPONENT_SYSTEM_PHASE` left in the codebase would be in the changelog, which raises the question: should this change be also added as a changelog entry?

Reply:
🤔 Not sure @gziolo ?